pkg/relabel/relabel.go

```diff
@@ -102,6 +102,87 @@ func relabel(lset labels.Labels, cfg *config.RelabelConfig) labels.Labels {
 	return lb.Labels()
 }
 
+// ProcessSet returns a relabeled copy of the given label set. The relabel configurations
+// are applied in order of input.
+// If a label set is dropped, nil is returned.
+// May return the input labelSet modified.
+func ProcessSet(labels model.LabelSet, cfgs ...*config.RelabelConfig) model.LabelSet {
```

Comment:
We should need only one of these two functions, we should be moving away from model.

Reply:
refactoring should need quite a bit of code changes so I thought to leave it for another PR.

Reply:
in general I think I will have time to do the refactoring , but I prefer to do it in few smaller PRs for easy review and merging without the need of constant rebase with master.

Reply:
@brian-brazil is it ok to do this in few stages to keep the PRs small?

Reply:
It's fine to do it in stages, but this PR is not one of those stages.

Reply:
Yes, and then you can delete the older relabel package.